Bugzilla/Auth.pm

```diff
@@ -234,7 +234,6 @@ sub _handle_login_result {
   # to find account names by brute force)
   elsif ($fail_code == AUTH_LOGINFAILED or $fail_code == AUTH_NO_SUCH_USER) {
     my $remaining_attempts = MAX_LOGIN_ATTEMPTS - ($result->{failure_count} || 0);
-    Bugzilla->iprepd_report('bmo.username_password_mismatch', remote_ip());
```

Comment:
No `check_rate_limit` here?

Reply:
Yeah. Not needed as Bugzilla already has a built in method for locking out a user if they enter the wrong password more than 5  times. They are then not allowed to log in again for 30 mins. It is keyed off of IP address.